tests/unit/test_context.py

```diff
@@ -184,24 +182,18 @@ def test_function_adding():
     assert c.schema[c.schema_name].function_lists[0].name == "F"
     assert c.schema[c.schema_name].function_lists[0].parameters[0][0] == "x"
     assert (
-        c.schema[c.schema_name].function_lists[0].parameters[0][1].getSqlType()
-        == int_sql_type
-    )
-    assert (
-        c.schema[c.schema_name].function_lists[0].return_type.getSqlType()
-        == float_sql_type
+        str(c.schema[c.schema_name].function_lists[0].parameters[0][1])
```

Comment:
Not needed for this PR, but I wonder if it would be better to start being more explicit in our function registration examples to use `int32` or `int64` to avoid the platform specific ambiguity.